Formula/amp.rb

```diff
@@ -0,0 +1,36 @@
+class Amp < Formula
+  desc "Text editor for your terminal"
+  homepage "https://amp.rs"
+  url "https://github.com/jmacdonald/amp/archive/0.6.2.tar.gz"
+  sha256 "9279efcecdb743b8987fbedf281f569d84eaf42a0eee556c3447f3dc9c9dfe3b"
+  head "https://github.com/jmacdonald/amp.git"
+
+  depends_on "cmake" => :build
```

Comment:
I thought you only used cargo for the build, so why do we need this cmake dep?

Reply:
Because the dependency graph contains some other dependency that apparently uses it for compilation 🤷

Reply:
yeah, amp is using cmake underlyingly.

```
Last 15 lines from /Users/rchen/Library/Logs/Homebrew/amp/01.cargo:
running: "cmake" "/Users/rchen/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/onig_sys-68.0.1/oniguruma" "-DBUILD_SHARED_LIBS=OFF" "-DCMAKE_INSTALL_PREFIX=/private/tmp/amp-20200621-83707-856n8t/amp-0.6.2/target/release/build/onig_sys-6385270c2b3e2efc/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC --target=x86_64-apple-darwin" "-DCMAKE_C_COMPILER=/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC --target=x86_64-apple-darwin" "-DCMAKE_CXX_COMPILER=/usr/local/Homebrew/Library/Homebrew/shims/mac/super/clang++" "-DCMAKE_BUILD_TYPE=Release"
```